SSA/Projects/InstCombine/lean/g2004h11h22hMissedhandhfold.lean

```diff
@@ -0,0 +1,51 @@
+import SSA.Projects.InstCombine.lean.g2004h11h22hMissedhandhfold_proof
+import SSA.Projects.InstCombine.LLVM.PrettyEDSL
+import SSA.Projects.InstCombine.TacticAuto
+import SSA.Projects.InstCombine.LLVM.Semantics
+import Batteries.Data.BitVec
```

Comment:
We do not need batteries here. This likely can be dropped in a separate commit for AliveStatements as well.